src/app/shared/post-it-collections/post-it-collections.component.ts

```diff
@@ -290,4 +245,12 @@ export class PostItCollectionsComponent implements OnInit, AfterViewInit, OnDest
     return HashCodeGenerator.hashString(collection.id || collection.correlationId);
   }
 
+  public notifyOfError(message: string) {
+    this.store.dispatch(new NotificationsAction.Error({message}));
```

Comment:
You do not need to show notifications by dispatching store actions. You can do so directly. It is not a problem, you are just going around for no good reason. We have actions for them to be able to easily switch some error action in store to notification action.